docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
+
+* Availability check
+
+== Requirements for availability check
+
+* PostgreSQL is accessible on network level
+* it's possible to login to PostgreSQL
+* it's possible to execute queries 
+
+== Solutions
+
+PostgreSQL managed by Stackgres:
```

Comment:
I guess you mean `postgresql_exporter` by Stackgres.

This section is a bit barebone for my taste, most importantly what's missing:

* A proper description what this solution would include. The word `postgresql_exporter` does not come up once in this section
* What metric we'd be using as an uptime measure and why. The exporter provides a lot of metrics. What counts as down and what counts as up?